server/pbench/bin/index-pbench

```diff
@@ -763,9 +802,39 @@ class ToolData(object):
         return ts.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
 
     def _make_source_unified(self):
-        """Process each .csv file at the same time, reading one row from each,
-        unifying them into one record per column identifier when the
-        timestamps match."""
+        """Create one JSON document per identifier, per timestamp from
```

Comment:
Can you provide an example of an "identifier"?

Reply:
Never mind...

Reply:
Actually, it's probably a good idea to provide a concrete instance of an identifier (e.g. for iostat an identifier is a disk) , so the example below is not quite so abstract.

Reply:
Just updated with further documentation.

Reply:
Okay, a few more updates, but I have not addressed a more concrete identifier.  Perhaps referencing the gold files would be better?

Reply:
Maybe  - I just find it easier to understand what is going in in the general case, if I have  a mapping to a concrete example that I can fit into my head.

Reply:
So this header comment shows the nature of the algorithm, but not all the details.  The details of the JSON document structure are talked about on line 990.